README.md

```diff
@@ -53,9 +53,9 @@ Example structure:
 ```yaml
 projects:
   -
-    name: 'Project Name'
+    name: 'Test run {master}'
```

Comment:
Should this say `{branch_name}`, not master?

Reply:
it should say `Test run {branch}` 👍 well spotted